src/pages/UserHome.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import firebase from 'firebase';
+import {
+  Button,
+  Container,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    paddingLeft: theme.spacing(6),
+  },
+  bannerRoot: {
+    backgroundColor: 'white',
+    marginTop: theme.spacing(4),
+    minHeight: '225px',
+    paddingLeft: theme.spacing(3),
+    paddingTop: theme.spacing(4),
+  },
+  browseButton: {
+    marginTop: theme.spacing(3),
+  },
+  addButton: {
+    textAlign: 'right',
+  },
+  scholarshipsAdded: {
+    marginTop: theme.spacing(6),
+  },
+}));
+
+export default function UserHome() {
+  const classes = useStyles();
+  const user = firebase.auth().currentUser;
+
+  function scholarshipLink() {
+    window.location.href = '/scholarships';
+  }
+
+  function addScholarshipLink() {
+    window.location.href = '/scholarships/new';
+  }
+
+  return (
+    <Container className={classes.root}>
+      <Typography variant="h4" gutterBottom className={classes.rootText}>
+        Welcome {user.displayName}
+      </Typography>
+      <Grid container className={classes.bannerRoot}>
+        <Grid item sm={8} xs={12}>
+          <Typography variant="h5" gutterBottom>
+            Looking for scholarships?
+          </Typography>
+          <Button
+            style={{ margin: '25' }}
+            variant="outlined"
+            color="primary"
+            onClick={scholarshipLink}
+            className={classes.browseButton}>
+            Browse Scholarships
+          </Button>
+        </Grid>
+        <Grid item sm={4} xs={12}>
+          {/* Image here */}
+        </Grid>
+      </Grid>
+
+      <Grid container className={classes.scholarshipsAdded} spacing={2}>
+        <Grid item sm={9}>
+          <Typography variant="h5" component="h2" gutterBottom>
+            Scholarships Added
+          </Typography>
+        </Grid>
+        <Grid item sm={3} className={classes.addButton}>
+          <Button
+            style={{ margin: '25' }}
```

Comment:
I don't think this renders to anything since there are no units. Also a style class is preferred.

Reply:
ok

Reply:
sure